app/locales/cs.json

```diff
@@ -34,7 +34,8 @@
     "break": "přestávky",
     "nextBreakFollowing_0": "\nDalší přestávka po {{count}} mikropřestávce",
     "nextBreakFollowing_1": "\nDalší přestávka po {{count}} mikropřestávkách",
-    "nextBreakFollowing_2": "\nDalší přestávka po {{count}} mikropřestávkách"
+    "nextBreakFollowing_2": "\nDalší přestávka po {{count}} mikropřestávkách",
+    "notificationStatus": "In do not disturb mode"
```

Comment:
[Apple](https://support.apple.com/en-us/HT204321) calls this `Do Not Disturb`so we should do the same